_config.yml

```
@@ -1,2 +1,32 @@
+theme: jekyll-theme-slate
+
+name: Streamline Tools CI
+
+on: [push, pull_request]
+
+jobs:
+  quality-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)